src/plugins/imagery/components/ImageryView.vue

```diff
@@ -232,6 +233,7 @@ const ARROW_LEFT = 37;
 const SCROLL_LATENCY = 250;
 
 const ZOOM_SCALE_DEFAULT = 1;
+const SHOW_THUMBS_THRESHOLD_HEIGHT = 400;
```

Comment:
@charlesh88 Does this seem like a reasonable threshold for showing the thumbnails? Would be great to get your feedback.

Reply:
Taking a look now...